heron/examples/src/java/multilang/resources/storm.py

```diff
@@ -0,0 +1,257 @@
+# -*- coding: utf-8 -*-
```

Comment:
If you have written it - you could use Twitter copyright or if you have taken code from Storm repo, we could retain the Storm copyright.